### PR TITLE
p4: add page

### PR DESCRIPTION
**Pull request by Lightyagami1**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/p4.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# p4
+
+> Version Control System.
```
Comment (agnivade) [inline pages/common/p4.md]: Command line tool for the Perforce version control system.

File: pages/common/p4.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`p4 add`
+
+- Create or edit changelist description:
+
+`p4 change`
+
+- Open a file to edit:
+
+`p4 edit -c <change-number> <file-name>`
```
Comment (agnivade) [inline pages/common/p4.md]: Please use the token syntax(`{{}}`) and snake_case for the variables. Also, how does a user get the change_number ?

Comment (Lightyagami1) [reply pages/common/p4.md]: I have modified my fork.
p4 change command will generate the change-number

**Comment by agnivade** [comment]
Thanks. I am good with this except the new pages. Let us add that in a separate PR. Because when we add the new pages, we need to remove the equivalent commands from the main `p4` page.

**Comment by Lightyagami1** [comment]
How can I remove my other account from "license/cla" list, I don't have any email associated with that name ?

**Comment by agnivade** [comment]
Ah, so the first 2 commits are under "Saurav Kumar", but later ones are under "Lightyagami1". So you need to squash those commits under Lightyagami1. Probably you have mixed your work email and personal email.

**Comment by Lightyagami1** [comment]
Created a new PR, [https://github.com/tldr-pages/tldr/pull/2403](url)

**Pull request closed without merging**